packages/react/popover/src/Popover.tsx

```diff
@@ -44,27 +49,60 @@ type PopoverOwnProps = {
 const Popover: React.FC<PopoverOwnProps> = (props) => {
   const { children, open: openProp, defaultOpen, onOpenChange } = props;
   const triggerRef = React.useRef<HTMLButtonElement>(null);
+  const [hasCustomAnchor, setHasCustomAnchor] = React.useState(false);
```

Comment:
I wasn't sure what to name this state ... ideas? 🤔

Reply:
That reads fine to me.